docker/flv2mp4-docker-image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos:7
+
+MAINTAINER Hemani Katyal <hkatyal@redhat.com>
+
+RUN yum install epel-release -y && yum update -y && \ 
+    rpm --import http://li.nux.ro/download/nux/RPM-GPG-KEY-nux.ro && \
+    rpm -Uvh http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm && \
+    yum install ffmpeg ffmpeg-devel -y
+
+ADD flv2mp4.sh /usr/local/bin/
```

Comment:
ok let's do it in that way. @hemanik can you work on this? And then release the docker image? @bartoszmajsak do we have arquillian organization there?